### DISCUSSION TITLE
斷句以及單字的時間和字幕不準確

### DISCUSSION THREAD

**Discussion by shawnchiao** [ZuodaoTech/everyone-can-use-english#631] [General]
在聽讀練習中
斷句有得時候頭和尾巴會不見
字幕的時間跟音頻的時間常常對不上
所以選了某個字或某段話播出來完全是別的地方
這個bug如果修好的話我覺得這個app就超強的

**Comment by an-lee** [comment]
感谢反馈。

能具体举例讲讲怎么对不上吗？截图看看？或者这音频贴出来，我测试一下看？

**Reply by Jonnyning** [reply]
> 感谢反馈。
> 
> 能具体举例讲讲怎么对不上吗？截图看看？或者这音频贴出来，我测试一下看？

我也碰到了这个问题，比如图上这个时刻，实际上语音已经后一句话都讲完了，文字还停留在前一句话，进度不一致，但是几句话之后又会追上，同步与不同步交替出现。用本地模型 whisper 和 opneai api 分别尝试过，都有这个问题。
![截屏2024-06-30 上午3 38 30](https://github.com/ZuodaoTech/everyone-can-use-english/assets/132688105/567fa328-d149-43eb-a677-8646893ec0d6)

**Reply by an-lee** [reply]
是不是中间有音乐？

**Reply by Jonnyning** [reply]
> 是不是中间有音乐？

有背景笑声，就是情景喜剧的那种
